### PR TITLE
Implemented plot options in Line Plot dialog

### PR DESCRIPTION
**Pull request by derekagorhom**
fixes (partly) #7556
replaces PR #7814

@rdstern  this PR is ready for review

### CODE REVIEW

**Comment by berylwaswa** [comment]
The Plot options for the line plot are all enabled. And all the aesthetics and geom parameters are in place. 
The first option is the Plot Options and this is followed by each of the remaining options for that dialogue, but they are not in alphabetical order. @derekagorhom I think if you arrange them in alphabetical order it'll be okay.

**Comment by berylwaswa** [comment]
The options are now well arranged in alphabetical order. @lloyddewit this looks fine now.

File: instat/dlgLinePlot.vb [hunk 1]
```diff
@@ -730,4 +784,42 @@ Public Class dlgLinePlot
...
+                                clsNewXLabsTitleFunction:=clsXlabsFunction, clsNewYLabTitleFunction:=clsYlabFunction, clsNewLabsFunction:=clsLabsFunction, clsNewFacetFunction:=clsRFacetFunction,
+                                clsNewThemeFunction:=clsThemeFunction, dctNewThemeFunctions:=dctThemeFunctions, clsNewGlobalAesFunction:=clsRaesFunction, ucrNewBaseSelector:=ucrLinePlotSelector,
+                                clsNewCoordPolarFunction:=clsCoordPolarFunction, clsNewCoordPolarStartOperator:=clsCoordPolarStartOperator, clsNewXScaleDateFunction:=clsXScaleDateFunction, clsNewAnnotateFunction:=clsAnnotateFunction,
+                                clsNewScaleFillViridisFunction:=clsScaleFillViridisFunction, clsNewScaleColourViridisFunction:=clsScaleColourViridisFunction, clsNewYScaleDateFunction:=clsYScaleDateFunction,
+                                strMainDialogGeomParameterNames:=strGeomParameterNames, bReset:=bResetSubdialog)
+        sdgPlots.ShowDialog()
+        bResetSubdialog = False
+    End Sub
+    Private Sub PeakOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles PeakOptionsToolStripMenuItem.Click
```
Comment (lloyddewit) [inline instat/dlgLinePlot.vb:797]:
```suggestion
    End Sub
    
    Private Sub PeakOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles PeakOptionsToolStripMenuItem.Click

```

**Comment by derekagorhom** [comment]
@lloyddewit i have made the requested changes

File: instat/dlgLinePlot.vb [hunk 2]
```diff
@@ -730,4 +740,44 @@ Public Class dlgLinePlot
...
+                                clsNewScaleFillViridisFunction:=clsScaleFillViridisFunction, clsNewScaleColourViridisFunction:=clsScaleColourViridisFunction, clsNewYScaleDateFunction:=clsYScaleDateFunction,
+                                strMainDialogGeomParameterNames:=strGeomParameterNames, bReset:=bResetSubdialog)
+        sdgPlots.ShowDialog()
+        bResetSubdialog = False
+    End Sub
+
+    Private Sub PeakOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles PeakOptionsToolStripMenuItem.Click
+        openSdgLayerOptions(clsPeakFunction)
+    End Sub
+    Private Sub DumbbellOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles DumbbellOptionsToolStripMenuItem.Click
```
Comment (lloyddewit) [inline instat/dlgLinePlot.vb:758]:
```suggestion
    End Sub
    
    Private Sub DumbbellOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles DumbbellOptionsToolStripMenuItem.Click
```

File: instat/dlgLinePlot.vb [hunk 3]
```diff
@@ -730,4 +740,44 @@ Public Class dlgLinePlot
...
+        openSdgLayerOptions(clsGeomDumbbellFunction)
+    End Sub
+
+    Private Sub LineOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles LineOptionsToolStripMenuItem.Click
+        openSdgLayerOptions(clsGeomLineFunction)
+    End Sub
+    Private Sub SmoothOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles SmoothOptionsToolStripMenuItem.Click
```
Comment (lloyddewit) [inline instat/dlgLinePlot.vb:765]:
```suggestion
    End Sub
    
    Private Sub SmoothOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles SmoothOptionsToolStripMenuItem.Click
```

File: instat/dlgLinePlot.vb [hunk 4]
```diff
@@ -730,4 +740,44 @@ Public Class dlgLinePlot
...
+    End Sub
+
+    Private Sub PointOptionToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles PointOptionToolStripMenuItem.Click
+        openSdgLayerOptions(clsPointsFunction)
+    End Sub
+
+    Private Sub PathOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles PathOptionsToolStripMenuItem.Click
+        openSdgLayerOptions(clsPathFunction)
+    End Sub
+    Private Sub StepOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles StepOptionsToolStripMenuItem.Click
```
Comment (lloyddewit) [inline instat/dlgLinePlot.vb:776]:
```suggestion
    End Sub
    
    Private Sub StepOptionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles StepOptionsToolStripMenuItem.Click
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@berylwaswa We made a few changes since your last approval. If you can retest/approve, then we can merge, thanks

**Comment by berylwaswa** [comment]
@derekagorhom @lloyddewit this looks okay.

**Review by berylwaswa — APPROVED** [review]
Looks good.

**Pull request merged**